SQF/dayz_code/stringtable.xml

```diff
@@ -5313,6 +5361,10 @@
 			<Russian>Армейский котелок для хранения жидкостей и еды.</Russian>
 			<German>Eine stabile Feldflasche, um Wasser zu transportieren.</German>
 		</Key>
+		<Key ID="STR_ITEM_CANTEEN_EMPTY">
+			<English>Empty</English>
```

Comment:
Can we leave those strings as different? They a bit different in russian. I think it's not a big deal.